_posts/2020-03-02-junior-dev-advice.markdown

```diff
@@ -0,0 +1,117 @@
+---
+layout: post
+title: "Some Resources and Advice For Junior Developers"
+date: 2020-03-02 14:30:00
+categories: []
+author: rdormer
+
+---
+
+# Tough Love
+
+One of the first things that a journeyman programmer will have to
+learn when they transition from full time student to working developer
+is that things in the real world are *never* as cut and dry as their
+classes may have made it seem.  At the end of the day, software is not
+written because we love building castles of logic in the sky, it's
+written to make money.  That might seem like a trite observation, but
+it explains almost everything you observe as a working developer.
+Because perfection isn't profitable, real world applications are
+developed under less than ideal circumstances, often under intense
+pressure, and with inevitable compromises.  Large applications sprawl,
+get messy, and develop stagnant corners and scary basements.
+Questionable or just plain puzzling decisions are made by developers
+without explanation, justification, or documentation by developers who
+then leave the company, never to be seen or heard from again.  Most
+junior developers are scared of making mistakes - imagine their
+surprise when they find themselves working on something that was
+written by another junior developer who did, in fact, make some major
+mistakes - mistakes that are now their responsibility to fix.
```

Comment:
@rdormer You're missing the `<!--more-->` tag. We need it so it doesn't display the entire article in the index page. Maybe you can break this first paragraph with it.